src/CommunityToolkit.Maui/Behaviors/PlatformBehaviors/IconTintColor/IconTintColorBehavior.macios.cs

```diff
@@ -119,4 +106,20 @@ static void SetUIImageViewTintColor(UIImageView imageView, View element, Color c
 		imageView.TintColor = color.ToPlatform();
 	}
 
+	void OnElementPropertyChanged(object? sender, PropertyChangedEventArgs e)
```

Comment:
we already subscribe to the `bindable.PropertyChanged` on base, does make sense to use the override method instead?

Reply:
Great catch! Done  ✅